apps/expo/src/components/header.tsx

```diff
@@ -0,0 +1,41 @@
+import { type ReactNode } from "react";
+import { Text, TouchableOpacity } from "react-native";
+import { Image } from "expo-image";
+import { useRouter } from "expo-router";
+import { Entypo, Ionicons } from "@expo/vector-icons";
+import { useUser } from "@supabase/auth-helpers-react";
+
+export function AuthAvatar() {
+  const user = useUser();
```

Comment:
maybe query from db and return only what we need, maybe from the Profile table?

Reply:
I think this is totally fine.